src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -563,6 +593,10 @@ export function useSwapInputsController({
     autoStart: false,
   });
 
+  if (internalSelectedInputAsset.value && internalSelectedOutputAsset.value) {
+    fetchQuoteAndAssetPrices();
+  }
+
```

Comment:
i don't think this is necessary since it should be handled via a `useAnimatedReaction`.